html/index.html

```diff
@@ -1,17 +1,337 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
+
+	<title>Document</title>
 </head>
 <body>
-    <div class="header">
-        <div class="wrapper">
-            <div class="logo"></div>
-            <div class="title">Helo</div>
+<div class="header">
+	<div class="wrapper">
+		<div class="header-container">
+			<div class="logo">
+				<a href="" class="header-icon">
+					<i class="fa fa-facebook" aria-hidden="true"></i>
+				</a>
+				<a href="" class="header-icon">
+					<i class="fa fa-pinterest" aria-hidden="true"></i>
+				</a>
+				<a href="" class="header-icon">
+					<i class="fa fa-twitter" aria-hidden="true"></i>
+				</a>
+				<a href="" class="header-icon">
+					<i class="fa fa-linkedin" aria-hidden="true"></i>
+				</a>
+			</div>
+			<div class="buttons">
+				<button>Eng</button>
+				<button>My Account</button>
+			</div>
+		</div>
+	</div>
+</div>
+<div class="wrapper">
+	<div class="banner-container">
+		<div class="preloader">
+			<img src="images/logo.png" alt="">
+		</div>
+		<div class="menu">
+			<ul>
+				<li>
+					<a class="" href="">HOME </a>
```

Comment:
Remove extra space after Home.
Change the HOME to lowercase letters and use `text-transform:uppercase` to make it uppercase using css.

If there is not class name , remove `class=""`